### PR TITLE
Temp page

### PR DESCRIPTION
**Pull request by cchave17**
Made some changes

### CODE REVIEW

File: tatted_dream/server/email/message.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+    def __init__(self, user_info):
+        msg_info = json.dumps(user_info)
+        load_msg = json.loads(msg_info)
+
+        # form information
+        self.name = load_msg["f_name"]
+        self.last_name = load_msg["l_name"]
+        self.travel_from = load_msg["travel_from"]
+        self.email = load_msg["email"]
+        self.phone = load_msg["phone"]
```
Comment (oldcycvez) [inline tatted_dream/server/email/message.py:15]: Hey what does this do

File: tatted_dream/server/email/send_email.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+import base64
+import os
+from email.mime.multipart import MIMEMultipart
+from email.mime.text import MIMEText
+
+CLIENT_SECRET_FILE = 'D:\Personal-Projects\Tatted-Dream\client_secret.json'
+API_NAME = 'gmail'
+API_VERSION = 'v1'
+SCOPES = ['https://mail.google.com/']
+email = os.environ["email"]
```
Comment (ebisa99) [inline tatted_dream/server/email/send_email.py:11]: trying to understand this part of the program. where are the modules being imported from? also testing my first pull request review

**Review by ebisa99 — COMMENTED** [review]
This is a test review.

File: tatted_dream/server/email/send_email.py (hunk 2 above)
Comment (cchave17) [reply tatted_dream/server/email/send_email.py:11]: So In java you use the pom.xml File to import all your dependencies. 

In Python, you import all your dependencies and libraries in the "requirments.txt" file.
if you open the requirements you will see most of these modules in there. 
HOWEVER, If you notice on line 1, I'm importing a Class from a different package, or directory within this same project.

**Pull request closed without merging**